install/alpine-wireguard-install.sh

```diff
@@ -41,32 +41,33 @@ PostUp = iptables -A FORWARD -i wg0 -j ACCEPT; iptables -A FORWARD -o wg0 -j ACC
 PostDown = iptables -D FORWARD -i wg0 -j ACCEPT; iptables -D FORWARD -o wg0 -j ACCEPT; iptables -t nat -D POSTROUTING -o eth0 -j MASQUERADE;
 ListenPort = 51820
 EOF
+$STD echo "net.ipv4.ip_forward=1" >>/etc/sysctl.conf
```

Comment:
```suggestion
echo "net.ipv4.ip_forward=1" >>/etc/sysctl.conf
```
yea, leave it without $STD